google/cloud/spanner_v1/database.py

```diff
@@ -155,6 +160,8 @@ def __init__(
         self._encryption_config = encryption_config
         self._database_dialect = database_dialect
         self._database_role = database_role
+        self._enable_drop_protection = enable_drop_protection
+        self._reconciling = False
```

Comment:
I can't see any place where this value is updated?

Reply:
This is updated at line 359 of this file, using the property setter.

Reply:
I am referring to `reconciling`